EPFParser.py

```diff
@@ -177,8 +180,11 @@ def seekToRecord(self, recordNum):
 
         Seeks to the beginning of the file if recordNum <=0,
         or the end if it's greater than the number of records.
+
+        N.B. with tbz streams, "0" is actually at 512.
         """
-        self.seekPos = 0
+        if self.seekPos != 512:
```

Comment:
Feels like this 512 value should be a constant, which would make the code a bit more self-explanatory.
e.g. TAR_HEADER_LENGTH (or similar?)

Reply:
yeah, that can be done.

Reply:
done in 3eeb076